src/main/java/com/example/bookstoreapp/dto/CreateBookRequestDto.java

```diff
@@ -0,0 +1,16 @@
+package com.example.bookstoreapp.dto;
+
+import java.math.BigDecimal;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class CreateBookRequestDto {
```

Comment:
same here.
Also, this DTO could be used in an update request as well so I'd rename it to just `BookRequestDto`, or we can do this later on, it's up to you, just make sure not to create another class for update when we can reuse this one 😉

Reply:
> same here. Also, this DTO could be used in an update request as well so I'd rename it to just `BookRequestDto`, or we can do this later on, it's up to you, just make sure not to create another class for update when we can reuse this one 😉

I will keep in mind. And will not forget